Client/src/Components/Authentication/AuthenticationValidationSchema.tsx

```diff
@@ -28,4 +28,21 @@ export const loginValidationSchema = Yup.object().shape({
     .required(requiredMessage("Email Address")),
   password: Yup.string()
     .required(requiredMessage("Password"))
+})
+
+export const forgotPasswordValidationSchema = Yup.object().shape({
+  email: Yup.string()
+    .email()
+    .required(requiredMessage("Email Address"))
+})
+
+export const resetPasswordValidationSchema = Yup.object().shape({
+  password: Yup.string()
+    .required(requiredMessage("Password"))
+    .min(8, "Password is short, should be 8 characters minimum"),
+  passwordConfirmation: Yup.string()
+    .required(requiredMessage("Confirm Password"))
+    .min(8, "Password is short, should be 8 characters minimum")
+    .oneOf([Yup.ref('password'), null], 'Passwords must match'),
+  resetToken: Yup.string()
```

Comment:
i assume the resetToken is not part of the validation schema anymore?

Reply:
Yes you're right